app/src/main/java/io/github/droidkaigi/confsched2017/receiver/NotificationReceiver.java

```diff
@@ -39,7 +37,6 @@ public void onReceive(Context context, Intent intent) {
             Timber.tag(TAG).v("Notification is disabled.");
             return;
         }
-        int sessionId = intent.getIntExtra(KEY_SESSION_ID, 0);
```

Comment:
Created the issue! Thanks!
https://github.com/DroidKaigi/conference-app-2017/issues/151